content/developer/rom/dossier/respondents.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Respondents
+status: draft
+---
+
+* TOC
+{:toc}
+
+Respondents are people that fill out questionnaires. The default respondent is the patient, but there's also profess, parent, second_parent, teacher and caregiver.
+
+- In the future second_parent will be merged into parent.
+- You won't see a profess respondent, since you can't send out to professionals, since there isn't usually just one.
```

Comment:
Klopt, maar dat is voor een enkel  medo project, waar dat heel specifiek mogelijk is en die weten het dan ook. Daarbij doen wij alles voor medo. 
Voor de gewone klant bestaat het niet, dus leek me niet nuttig ze hier te gaan verwarren.